flotta-agent.spec

```diff
@@ -73,28 +57,27 @@ install ./bin/device-worker %{buildroot}%{_libexecdir}/yggdrasil/device-worker
 install ./bin/device-worker-race %{buildroot}%{_libexecdir}/yggdrasil/device-worker-race
 make install-worker-config USER=%{flotta_user} HOME=/home/%{flotta_user} LIBEXECDIR=%{_libexecdir} BUILDROOT=%{buildroot} SYSCONFDIR=%{_sysconfdir}
 
+install -Dpm 644 %{flotta_systemd}/flotta-agent.sysusers %{buildroot}%{_sysusersdir}/%{name}.conf
+install -Dpm 644 %{flotta_systemd}/flotta.conf %{buildroot}/etc/tmpfiles.d/%{name}.conf
+
 %files
 %{_libexecdir}/yggdrasil/device-worker
 %{_sysconfdir}/yggdrasil/
-%dir %attr(0755, %{flotta_user}, %{flotta_user}) %{_sysconfdir}/yggdrasil/device/volumes
+%{_sysusersdir}/%{name}.conf
+/etc/tmpfiles.d/%{name}.conf
 
 %files race
 %{_libexecdir}/yggdrasil/device-worker-race
 %{_sysconfdir}/yggdrasil/
-%dir %attr(0755, %{flotta_user}, %{flotta_user}) %{_sysconfdir}/yggdrasil/device/volumes
+%{_sysusersdir}/%{name}.conf
+/etc/tmpfiles.d/%{name}.conf
 
 %post race
-loginctl enable-linger %{flotta_user}
-
-# HACK till https://bugzilla.redhat.com/show_bug.cgi?id=2060702 is fixed
-setenforce 0
-systemctl enable --now --machine %{flotta_user}@.host --user podman.socket
-setenforce 1
-# END HACK
-
-systemctl enable --now nftables.service
```

Comment:
What about the nftables? I can't see anywhere it to be enabled?

Reply:
On rpm-ostree is doing anything at all, but I keep it for non ostree base os.